### ISSUE TITLE
CeleryPingHealthCheck: Add 'limit' argument for ping

### ISSUE THREAD

**Issue by cbwchuck** [codingjoe/django-health-check#353]
CeleryPingHealthCheck makes use of `celery.app.control.ping` with `timeout` argument set. 

But celery always wait for `timeout` seconds before returning the responses, which is a bit counter-intuitive.

If what we want here is to return the ping result immediately after receiving the responses, we will need to specify `limit=<num_of_celery_workers_expect_to_see>` as documented in [Celery's documentation](https://docs.celeryq.dev/en/stable/reference/celery.app.control.html?highlight=ping#celery.app.control.Control.broadcast).

**Comment by codingjoe** [comment]
Hi @cbwchuck 👋,

Thanks for the late reply. Sounds sane. What precisely did you have in mind? Don't hesitate to draft a PR to get the ball rolling.

Cheers!
Joe

**Comment by codingjoe** [comment]
@cbwchuck I tried setting a limit; however, I don't see a measurable improvement to the response time. Can you check out the branch and leave me some feedback? Thanks! Joe

**Comment by codingjoe** [comment]
@skarzi, @lociii, @kevinmarsh, @amrishparmar and @samastur any feedback from your side maybe? I'd love to include it in v4, if it make sense. But I am struggling to see any effect myself.

**Comment by lociii** [comment]
@codingjoe I tested with a single worker active.

`app.control.ping(timeout=10, limit=1)` returns right after it received a response from that one worker. 
```
1770713412.94386
[{'celery@5898fa1d341e': {'ok': 'pong'}}]
1770713412.976099
```

`app.control.ping(timeout=10, limit=5)` only returns after the 10 second timeout.
```
1770713447.1909597
[{'celery@5898fa1d341e': {'ok': 'pong'}}]
1770713457.2432978
```

**Comment by codingjoe** [comment]
@lociii yeah 🎸 those are the benchmarks I was looking for. Let's ship it 👍
